src/Http/Requests/DeleteMediaRequest.php

```diff
@@ -24,6 +25,10 @@ public function validateResolved(): void
             throw new NotFoundHttpException(__('validation.exceptions.not_found', ['entity' => 'Media']));
         }
 
+        if (str_contains($this->media->name, 'preview')) {
```

Comment:
let's check that the media is preview by searching media with the preview_id field equals to the requested id